### DISCUSSION TITLE
Instructions on deploying Superset to AWS ECS

### DISCUSSION THREAD

**Discussion by ahaffar** [apache/superset#18348] [Q&A / Help]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
We deployed superset on AWS ECS using the official superset DockerHub image that is placed behind ALB, all is working fine so far, My question though is related if there is a need to set up AWS EFS which is an Elastic File System that will be used to permanently save any required data by superset such as configurations, settings in case the ECS containers rotate/reboot.

Our setup hosts the Postgres DB and REDIS Broker on Aurora/ElastiCache, I am looking if there is any data that needs to be saved on the container that runs Superset in case of the container restarted/spinning up a new container.

**Comment by ahaffar** [comment]
hello all
any advise here

**Reply by 1c7** [reply]
Why there are no update for 2 month (Dec 2021 -> Feb 2022)
I am having the same question too.  
**Superset should have a detail document about how to deploy in a production environment**

I just about to open a new issue and I saw this discussions

**Reply by mdeshmu** [reply]
Guys, I am also looking for AWS ECS + Superset documentation.

**Reply by bethany-poulin** [reply]
I agree with you completely, an official cloud formation or terraform script with explanations of how it works would be amazing!

**Comment by igamat** [comment]
I am also interested in installing Superset in AWS ECS with an AWS EFS. Are there any updates on this topic ? Has anyone been able to find any documentation regarding this ? Is this something feasible @villebro @rusackas @geido ? Thanks

**Reply by geido** [reply]
I don't think we have documentation about this specifically. If you are willing to help setting this up, feel free to reach out on [Superset Slack](https://join.slack.com/t/apache-superset/shared_invite/zt-2be0drwz8-bxPfkdz28ozzk1Iox29ufg).

**Reply by nayanmshah** [reply]
Any updates on this?

**Reply by rusackas** [reply]
Seems like everyone deploys Superset differently. If people want to provide documentation updates about the ways in which they do it so others can follow their pattern, that'd be great. Preset, for example, deploys in AWS, but we do it in our own way that's not what you're after here. As @geido mentioned, you can take this to the `#deploying-superset` channel on Slack for help, and search around for people and threads there where this has been discussed previously.
